docs/actions.md

```diff
@@ -392,35 +392,34 @@ const projects = await flowResult(store.fetchProjects())
 
 <!--END_DOCUSAURUS_CODE_TABS-->
 
-## Using flow instead of async / await {🚀}
+## async·await 대신에 flow 사용하기 {🚀}
 
-Usage:
+사용 방법:
 
--   `flow` _(annotation)_
+-   `flow` _주석(annotation)_
 -   `flow(function* (args) { })`
 
-The `flow` wrapper is an optional alternative to `async` / `await` that makes it easier to
-work with MobX actions.
-`flow` takes a [generator function](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Generator) as its only input.
-Inside the generator, you can chain promises by yielding them (instead of `await somePromise` you write `yield somePromise`).
-The flow mechanism will then make sure the generator either continues or throws when a yielded promise resolves.
+`flow` 래퍼는 MobX action을 더 쉽게 작업할 수 있게 `async`·`await` 대신 사용할 수 있는 옵션입니다.
+`flow`는 [generator function](https://developer.mozilla.org/ko/docs/Web/JavaScript/Reference/Global_Objects/Generator)을 유일한 입력으로 사용합니다.
+generator 안에서 yield를 사용하여(`await somePromise` 대신 `yield somePromise`) promise를 연결할 수 있습니다.
+flow 매커니즘은 promise가 해결될 때 generator가 계속 진행되거나 throw되는지 확인합니다.
 
-So `flow` is an alternative to `async` / `await` that doesn't need any further `action` wrapping. It can be applied as follows:
+따라서 `flow`는 `async`·`await`과 달리 `action` 래핑이 필요하지 않습니다. 이는 다음과 같이 적용될 수 있습니다.
 
-1. Wrap `flow` around your asynchronous function.
-2. Instead of `async` use `function *`.
-3. Instead of `await` use `yield`.
+1. 비동기 함수를 `flow`로 감쌉니다.
+2. `async` 대신  `function *`을 사용합니다.
+3. `await` 대신 `yield`를 사용합니다.
 
-The [`flow` + generator function](#asynchronous-actions) example above shows what this looks like in practice.
+[`flow` + generator function](#asynchronous-actions) 예제에서 실제로 어떻게 사용하는지 확인해보세요.
 
-Note that the `flowResult` function is only needed when using TypeScript.
-Since decorating a method with `flow`, it will wrap the returned generator in a promise.
-However, TypeScript isn't aware of that transformation, so `flowResult` will make sure that TypeScript is aware of that type change.
+`flowResult` 함수는 TypeScript를 사용할 때만 필요합니다.
+메서드를 `flow`로 데코레이팅하기 때문에 반환된 generator는 promise로 래핑 됩니다.
+그러나 TypeScript는 이러한 변환을 인식하지 못하기 때문에 `flowResult`를 사용하여 해당 변경을 인식할 수 있도록 해야 합니다.
 
-`makeAutoObservable` and friends will automatically infer generators to be `flow`s. `flow` annotated members will be non-enumerable.
```

Comment:
@iChenLei Could you tell me in detail what friends mean?

Reply:
@mweststrate Sir, look here.

Reply:
@mweststrate Sir, look here.

Reply:
similar api's that create observable fields, e.g. `makeObservable`, `extendObservable`, `observable(object)` etc.

Reply:
Thank you. I understand. :)